rts.tests/svscan-sigterm.sh

```diff
@@ -122,7 +122,7 @@ echo
 check_pid_sanity() {
   if [ `echo $1 | grep -E '^[1-9][0-9]{0,4}$' | wc -l` != "1" ] \
     || [ $1 -le 1 ]                                             \
-    || [ $1 -ge 65536 ]
+    || [ $1 -ge 99999 ]
```

Comment:
On Linux with cgroups, the PID could potentially be 32-bits. I don't know if checking for a maximum value adds a lot here.